libs/martech/martech.js

```diff
@@ -190,8 +182,23 @@ const setupEntitlementCallback = () => {
   );
 };
 
+function isProxied() {
+  // return window.location.hostname.endsWith('adobe.com');
+  const { hostname } = window.location;
+  return (
+    hostname === 'www.adobe.com'
+    || hostname === 'www.stage.adobe.com'
+    || hostname === 'milo.adobe.com'
+    || hostname === 'milo.stage.adobe.com'
+    || hostname === 'business.adobe.com'
+    || hostname === 'business.stage.adobe.com'
+    || hostname === 'blog.adobe.com'
+    || hostname === 'blog.stage.adobe.com'
+  );
+}
+
```

Comment:
This could be shorter
```suggestion
const isProxied = () => [
    'www.adobe.com', 'www.stage.adobe.com', 'milo.adobe.com',
    'milo.stage.adobe.com', 'business.adobe.com', 
    'business.stage.adobe.com', 'blog.adobe.com', 'blog.stage.adobe.com'
  ].includes(window.location.hostname)
```